patches/@ecl-twig/ec-component-footer-standardised+2.33.2.patch

```diff
@@ -0,0 +1,11 @@
+patch-package
```

Comment:
@drishu I don't think this patch is needed. I checked on my local without applying the patch and ecl-icon--xs class is applied by ecl-twig. Can you please check again without the patch?

Reply:
Nvm. It's needed.